modules/source/mongodb/config/mongodb.xml

```diff
@@ -17,9 +19,15 @@
 		username="${username}" password="${password}" />
 
 	<int-mongodb:inbound-channel-adapter
-		id="inbound" channel="output" query="${query}" collection-name="${collectionName}">
+		id="inbound" channel="aggregates" query="${query}" collection-name="${collectionName}">
 		<int:poller fixed-rate="${pollRate}" max-messages-per-poll="${maxMessages}" />
```

Comment:
While not really a "documentation" issue, we should use `fixedDelay` not `pollRate` in this source to be consistent with the `time` and `trigger` sources.

For the same reason, we should also consider adding `initialDelay` and `timeUnit` - see https://github.com/spring-projects/spring-xd/pull/1384/files and the `PeriodicTriggerMixin`.